terms.html

```diff
@@ -113,7 +113,7 @@
   <li>creates and maintains (<a>authors</a>) data, including <a>identifiers</a>, <a>claims</a>, and <a>verifiable credentials</a></li>
   <li>decides on the semantics of all data that it <a>authors</a> (i.e., what the data means)</li>
   <li>gathers, stores, processes, and disseminates data</li>
-  <li>manages its own logic, deciding what is, and what is not, true, valid, trustworthy, what to reuse (e.g., in terms of shared vocabularies and contexts that other <a>parties</a> have produced), etc.
+  <li>manages its own logic, deciding what is, and what is not, true, valid, trustworthy, what to reuse (e.g., in terms of shared vocabularies and contexts that other <a>parties</a> have produced), etc.</li>
 </ol>
```

Comment:
This list was begun on line 112, as `<ul>`. It must therefore be *closed* with `</ul>`, not `</ol>`.
```suggestion
</ul>
```